.drone.star

```diff
@@ -533,7 +574,143 @@ config = {
     "build": True,
 }
 
+# UI Test Suites
+# These list contains all the test suites that are present
+# When adding new test suites, make sure to update these lists
+federatedTestSuites = [
+    "webUISharingExternal",
+]
+
+federatedRootTestSuites = [
+    "webUISharingExternalToRoot",
+]
+
+rootSharingTestSuites = [
+    "webUIResharingToRoot",
+    "webUISharingAcceptSharesToRoot",
+    "webUISharingInternalGroupsToRoot",
+    "webUISharingInternalGroupsToRootEdgeCases",
+    "webUISharingInternalGroupsToRootSharingIndicator",
+    "webUISharingInternalUsersExpireToRoot",
+    "webUISharingInternalUsersToRoot",
+    "webUISharingInternalUsersToRootBlacklisted",
+    "webUISharingInternalUsersToRootCollaborator",
+    "webUISharingInternalUsersToRootPreviews",
+    "webUISharingInternalUsersToRootShareWithPage",
+    "webUISharingInternalUsersToRootSharingIndicator",
+    "webUISharingPermissionToRoot",
+]
+
+notificationsTestSuite = [
```

Comment:
```suggestion
notificationsTestSuites = [
```
POLA - I am astonished to see multiple items in this array, but no "s" at the end of the name ;)

Reply:
Done